lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -355,7 +355,7 @@ function dosomething_campaign_preprocess_pitch_page(&$vars, &$wrapper) {
   $vars['tagline'] .= t('Any cause, anytime, anywhere.');
 
   // Get sign up total and add it to cta
```

Comment:
Picky picky, but can we change this to help make the code more readable? This is not signup total.  Might be nice to update the comment with a description of why this is happening.